docs/merkletree/treetex/main.go

```diff
@@ -96,13 +99,14 @@ var (
 
 // nodeInfo represents the style to be applied to a tree node.
 type nodeInfo struct {
-	incProof     bool
-	incPath      bool
-	target       bool
-	perfectRoot  bool
-	ephemeral    bool
-	leaf         bool
-	rangeIndices []int
+	proof             bool
+	incPath           bool
+	target            bool
+	perfectRoot       bool
+	ephemeral         bool
+	leaf              bool
+	rangeIndices      []int
+	proofRangeIndices []int
```

Comment:
It's not really a proof. A range proof is what **complements** a compact range.

How about renaming `rangeIndices` to `dataRangeIndices`, and `proofRangeIndices` to `rangeIndices`?

Reply:
done